docs/release-notes/9.0.asciidoc

```diff
@@ -4,29 +4,65 @@
 coming::[9.0.0]
 
 [discrete]
-[[release-notes-9.0.0]]
-=== 9.0.0
+[[release-notes-9.0.0-beta]]
+=== 9.0.0-beta
 
 [discrete]
-[[known-issue-9.0.0]]
+[[breaking-changes-9.0.0-beta]]
+==== Breaking changes
+* Removes deprecated bulk action API endpoints ({kibana-pull}207906[#207906]).
+* Refactors the Timeline HTTP API endpoints ({kibana-pull}200633[#200633]).
+* Removes deprecated API endpoints for {elastic-defend} ({kibana-pull}199598[#199598]).
+* Disables deprecated API endpoints for bulk CRUD actions ({kibana-pull}197422[#197422]).
```

Comment:
```
* Removes deprecated bulk action API endpoints ({kibana-pull}207906[#207906]).
* Disables deprecated API endpoints for bulk CRUD actions ({kibana-pull}197422[#197422]).
```

These two are about the same thing and I'd suggest to make it a single release note:

```suggestion
==== Breaking changes
* Refactors the Timeline HTTP API endpoints ({kibana-pull}200633[#200633]).
* Removes deprecated API endpoints for {elastic-defend} ({kibana-pull}199598[#199598]).
* Removes deprecated API endpoints for bulk CRUD actions on detection rules ({kibana-pull}197422[#197422], {kibana-pull}207906[#207906]).
```